### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by PinhaEsquilo** [Ndragomirov/twitch-now#165]
- Twitch Now, has a memory leak, makes the laggy as hell.

- Twitch Now, don't save settings, i change interval refresh for 1 min but when i restart browser he don't save.(FIREFOX)
FIX PLEASE.

**Comment by SW1FT** [comment]
I advise you to use **at least** a 10 minute interval refresh.

**Comment by PinhaEsquilo** [comment]
I use Firefox 51.0.1(32-bit) Lastest version
SSD Samsung 850 PRO 256GB(Firefox is installed on it)
Corsair Vengeance 8GB DDR3
Intel-I5 4460 3.2GhZ.

1H using firefox with 6-7 tab. my memory has been around 2.3GB(laggy as fuck)
I uninstal this plugin only for test. and 1H it's been around 900MB~1.2GB.
(sorry for my bad englando =D )

**Comment by SW1FT** [comment]
Very weird. Could you try Firefox 64-bit with a new profile and install the same addons to see if it happens again?

**Comment by PinhaEsquilo** [comment]
Yeah, its happen with 64 bit too... unfortunately :'(... I only use twitch-live and ad-block... but I uninstall adblock just for test and happens same thing. Very weird!!!.

**Comment by Ndragomirov** [comment]
Current version of firefox addon is outdated, i will upload new version on next week
